x-pack/plugins/security_solution/public/detections/containers/detection_engine/rules/use_pre_packaged_rules.tsx

```diff
@@ -93,6 +114,11 @@ export const usePrePackagedRules = ({
             rulesNotInstalled: prePackagedRuleStatusResponse.rules_not_installed,
             rulesNotUpdated: prePackagedRuleStatusResponse.rules_not_updated,
           });
+          setTimelineStatus({
```

Comment:
Having two states for rules and timeline, we will re-render twice because we are updating two states